### DISCUSSION TITLE
One-Click: create corresponding relationship (reverse relationship) after setup

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#15191] [Feature Requests]
A lot of time a corresponding relation field makes managing data way easier. Setting those up while creating a relation is super nice and convenient with the one click button.

If the relation already exists, we need to setup the corresponding relation manually.  I'm curious if the one-click action also could be possible on in that case 🤔

![Bildschirmfoto 2022-08-20 um 18 54 36](https://user-images.githubusercontent.com/46492597/185758100-249127c2-b0b0-4b08-94ea-8680b38c4555.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
